pkg/terraform/state/deploy.go

```diff
@@ -472,7 +475,7 @@ func tryUpdate(f func() error) error {
 	for i := 0; i <= 3; i++ {
 		err := f()
 		if err != nil {
-			if k8sError.IsConflict(err) {
+			if k8sError.IsConflict(err) || k8sError.IsNotFound(err) {
```

Comment:
Drop the isnotfound check

Reply:
agreed.

Reply:
https://kubernetes.slack.com/archives/C0EG7JC6T/p1562794268483900 this is where this came from, for reference.